cmd/k8s-operator/operator_test.go

```diff
@@ -6,24 +6,15 @@
 package main
 
 import (
-	"context"
```

Comment:
All changes in this file correspond to moving functionality shared between ingress/egress proxy tests in this file and Connector tests to `cmd/k8s-operator/testutils_test.go`